sdk/storage/Azure.ResourceManager.Storage/tests/Tests/StorageAccountTests.cs

```diff
@@ -1187,11 +1187,15 @@ public async Task CreateStorageAccountWithEnableNfsV3()
 
         [Test]
         [RecordedTest]
+        [Ignore("Looks like a regression, temporarily removed")]
```

Comment:
This regression should be a corner case after introducing this [autorest change](https://github.com/Azure/autorest.csharp/pull/1838), so temporarily remove this test, will do more investigation.

Reply:
This should be caused by the swagger/server are not aligned on the DeletedAccounts API. 
Swagger/Server both owned by SRP team.
SRP team has confirmed in mail: 
> We have a work item dealing with swagger correctness and the highlighted issue will be addressed in the next api version release.